.gitignore

```diff
@@ -43,3 +43,8 @@ MuseScore_General.sf3
 MuseScore_General-License.md
 MuseScore_General-changelog.txt
 VERSION
+/buildInstallLog.txt
+/buildReleaseLog.txt
+/nbproject
+/*.stackdump
+/CMakeLists.txt
```

Comment:
changes to CMakeLists.txt should surly not get ignored

Reply:
I was having some trouble with my submission here.  I needed to modify CMakeLists.txt to get Musescore to compile on my machine.  The change here was an attempt to stop my local CMakeLists.txt from being pushed to the repository.  I do not intend to be changing CMakeLists.txt in central.  I'm new to git and not sure how to do this.

Reply:
Should not be needed, if you need to edit that file to get MuseScore to build, either everyone needs the change you your build envirioment is wrong somehow.